### PR TITLE
Display set of2d pts

### PR DESCRIPTION
**Pull request by kerautret**
# PR Description
new tool to display sets of 2d points

# Checklist

- [x] Doxygen documentation of the code completed (classes, methods, types, members...)
- [x] Check if it follows the tools structure described in [CONTRIBUTING.md](https://github.com/DGtal-team/DGtalTools-contrib/blob/master/CONTRIBUTING.md)
- [x] New entry in the [ChangeLog.md](https://github.com/DGtal-team/DGtalTools-contrib/blob/master/ChangeLog.md) added.
- [x] Update the readme with potentially a screenshot of the tools if it applies. 
- [x] No warning raised in Debug ```cmake``` mode (otherwise, Travis C.I. will fail).

### CODE REVIEW

**Comment by kerautret** [comment]
can we merge ?

File: visualisation/displaySet2dPts.cpp [hunk 1]
```diff
@@ -0,0 +1,252 @@
...
+///////////////////////////////////////////////////////////////////////////////
+namespace po = boost::program_options;
+
+
+/**
+ @page displayLineSegments displayLineSegments
+ 
+ @brief  Description of the tool...
+ 
+ @b Usage:   displayLineSegments [input]
```
Comment (dcoeurjo) [inline visualisation/displaySet2dPts.cpp]: Pts ?

File: visualisation/displaySet2dPts.cpp [hunk 2]
```diff
@@ -0,0 +1,252 @@
...
+ 
+ Typical use example:
+ ./visualisation/displaySetOf2dPts -i example.sdp  -o export.eps
+ 
+ @endcode
+ 
+ @b Example:
+ 
+ @code
+ displayLineSegments -i  $DGtal/examples/samples/....
```
Comment (dcoeurjo) [inline visualisation/displaySet2dPts.cpp]: Same gèrent

Comment (dcoeurjo) [reply visualisation/displaySet2dPts.cpp]: Here

File: visualisation/displaySet2dPts.cpp (hunk 1 above)
Comment (kerautret) [reply visualisation/displaySet2dPts.cpp]: yes thanks (due to name change)

File: visualisation/displaySet2dPts.cpp (hunk 2 above)
Comment (kerautret) [reply visualisation/displaySet2dPts.cpp]: thanks

**Comment by kerautret** [comment]
done

**Comment by kerautret** [comment]
merging (change done)

**Pull request merged**